### PR TITLE
Elasticsearch

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by amitaibu** [comment]
Looks good. Can you please add some info in the README as-well

**Comment by AronNovak** [comment]
@amitaibu Extended the doc.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,18 @@ The directory structure:
...
+The starter kit comes out of the box with ElasticSearch. Search API is activated and DDEV provides an ElasticSearch instance, already configured to use a [stopwords](https://github.com/Gizra/drupal-starter/blob/master/config/elasticsearch/stopwords.txt) and a [synonyms](https://github.com/Gizra/drupal-starter/blob/master/config/elasticsearch/synonyms.txt) list. Also it creates 4 indices (QA, DEV, TEST and LIVE) to reflect our typical Pantheon setup. The site inside DDEV will use the DEV index.
+To take a look, you can check these first:
+ - https://drupal-starter.ddev.site:9201/
+ - https://drupal-starter.ddev.site:9201/\_cat/indices - list of all indices
+ - https://drupal-starter.ddev.site:9201/\_search - list of all documents
+
+### Disable ElasticSearch
+
+In case the project doesn't require any search, you can easily drop ElasticSearch support by reverting https://patch-diff.githubusercontent.com/raw/Gizra/drupal-starter/pull/67.patch - see https://www.drupal.org/patch/reverse . As the development continues, it will be more and more conflicts when trying to revert the patch, but it clearly marks the key points that must be deleted.
+
```
Comment (amitaibu) [inline README.md:71]: I'm not sure we need it. Also, as time will pass, reverting this patch might prove to be harder.

Comment (AronNovak) [reply README.md:71]: Alright, if it needs to be dropped, it can be done anyways, deleting this part.

**Review by amitaibu — APPROVED** [review]

**Pull request merged**